app/services/nicolive-program/nicolive-program.ts

```diff
@@ -44,6 +50,9 @@ interface INicoliveProgramState extends ProgramState {
   isExtending: boolean;
   isStarting: boolean;
   isEnding: boolean;
+
+  // 永続化しない情報だが、ProgramStateにもたせたいためここに置く
+  serverClockOffset?: number; // in seconds
```

Comment:
変数名の方にも単位をつけておくとより間違いにくくなると思いますー

```suggestion
  serverClockOffsetSec?: number; // in seconds
```

Reply:
反映しましたー